content/docs/core/configuration.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Configuration
+description: This page discusses configuration settings for Pomerium Core.
+keywords: [core configuration]
+sidebar_label: Configuration
+---
+
+# Configuration
+
+You can configure Pomerium using either a configuration file or [environment variables](https://en.wikipedia.org/wiki/Environment_variable). If using a configuration file, the following languages are supported:
+
+- [YAML](https://yaml.org/)
+- [JSON](https://www.json.org/json-en.html)
+- [TOML](https://toml.io/en/)
+
+(Our documentation always assumes a YAML configuration file.)
+
+You can use both environment variables and a configuration file. If a particular option is set using both an environment variable and a config file key, the environment variable will take precedence.
+
+:::tip
+
+Pomerium can hot-reload route configuration details, authorization policy, certificates, and other proxy settings.
+
+:::
+
+### Configuration syntax
+
+Both configuration file keys and environment variables are case sensitive.
+
+Configuration file keys are always lowercase. Environment variables are identical to configuration file keys, except they are always uppercase.
+
+See the [Reference](/docs/reference) page for a comprehensive list of Pomerium's configuration settings.
+
+## All-in-One and Split-Service modes
+
+You can configure Pomerium using either All-in-One mode or Split-Service mode.
+
+### All-in-One mode
+
+All-in-One mode means a single Pomerium process runs all of the [four logical services](/docs/internals/architecture#component-level). You can use All-in-One mode when running Pomerium:
+
+- As a single system service or container, or
+- In a distributed environment where there are multiple processes that each handle separate [Pomerium services](/docs/internals/architecture#component-level).
```

Comment:
I don't understand what this is saying. In all-in-one mode the services are combined, not separate.